### PR TITLE
Add precommit with prettier

### PR DESCRIPTION
**Pull request by billyxs**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (billyxs) [inline .prettierrc]: Saw your prettier fixes entering the mix. I guess it is time.

File: package.json [hunk 2]
```diff
@@ -16,7 +16,16 @@
...
-  "author": "Billy Montgomery billy.montgomery@hixme.com",
+  "author": {
+    "name": "Billy Montgomery",
+    "email": "billy.montgomery@gmail.com",
+    "url": "https://github.com/billyxs"
+  },
+  "contributors": [{
+    "name": "Calvin Cieslak",
+    "url": "https://github.com/seesleestak"
+  }],
```
Comment (seesleestak) [inline package.json]: 😀

File: .prettierrc (hunk 1 above)
Comment (seesleestak) [reply .prettierrc]: 🤷🏼‍♂️

File: package.json [hunk 3]
```diff
@@ -76,8 +88,20 @@
     "lint:fix": "npm run lint -- --fix",
     "build": "cross-env BABEL_ENV=commonjs babel src --out-dir lib --source-maps",
     "flow": "flow",
+    "prettier": "prettier --write ./**/*.{js,css}",
```
Comment (seesleestak) [inline package.json]: Woah, does prettier format css too?

**Comment by seesleestak** [comment]
Oof, sorry for the merge conflicts

File: package.json (hunk 2 above)
Comment (billyxs) [reply package.json]: Feel free to update

Comment (seesleestak) [reply package.json]: Looks good to me. Thanks for adding it in :)

File: .prettierrc (hunk 1 above)
Comment (billyxs) [reply .prettierrc]: to semi or not to semi? I've been leaving semis in my newer projects as prettier has been taking care of them for me. doesn't matter to me.

Comment (seesleestak) [reply .prettierrc]: Honestly no preference for me. The projects/languages I've been spending time with have them, so I've kinda grown to expect it. And it's default behavior for prettier 🤷‍♂ 

Entirely up to you.

File: package.json (hunk 3 above)
Comment (billyxs) [reply package.json]: yeah, I guess it's not really applicable here. enabled it for markdown

**Pull request merged**